foxpuppet/browser/window.py

```diff
@@ -16,29 +18,42 @@ class BrowserWindow(object):
 
     def __init__(self, selenium, *args, **kwargs):
         self.selenium = selenium
-        self._windows = Windows(selenium)
```

Comment:
I have to pass a handle to it, and I don't see how I could pass in a handle before opening the new window. It would be the old handle. 

I could pass in the old one and then replace it just to satisfy the init.